library/network_connections.py

```diff
@@ -1010,23 +1007,71 @@ def delete_cb(connection, result, cb_args):
 
         # workaround libnm oddity. The connection may not yet be gone if the
         # connection was active and is deactivating. Wait.
-        wait_count = 0
+        c_uuid = connection.get_uuid()
+        gone = self.wait_till_connection_is_gone(c_uuid)
+        if not gone:
+            raise MyError(
+                "connection %s was supposedly deleted successfully, but it's still here"
+                % (c_uuid)
+            )
+
+    def volatilize_connection(self, connection, timeout=10):
+        update2_cb = Util.create_callback("update2_finish")
```

Comment:
this variable is only used at one place. I think it's preferable to just create and use the callback at the place, instead of assigning it first to another variable.

Reply:
> ```
> +        gone = self.wait_till_connection_is_gone(c_uuid)
> +        if not gone:
> ```
> same here. Do you think such variables help readability? I would think the contrary, but that's of course a matter of opinion.

To me it looks easier to read because it is smaller pieces to parse at a time.